scripts/generate_share_images/index.ts

```diff
@@ -0,0 +1,75 @@
+// You can run via `yarn generate-share-images`
```

Comment:
Q: Do we need to start the web application before running the script? Consider adding instructions if that's the case

Reply:
Good suggestion, done!